src/jio-footer.css

```diff
@@ -79,3 +83,8 @@ ul {
   margin-right: 2rem;
   padding: 1rem 0;
 }
+
+p .box {
```

Comment:
This will never match anything, there's no element with a class of box inside of a p tag

![image](https://user-images.githubusercontent.com/110087/213568783-13d8102c-6517-4661-bb0a-79260dc3ef6b.png)